### PR TITLE
[zh] Translate glossary dockershim

### PR DESCRIPTION
**Pull request by TinySong**

### CODE REVIEW

File: content/zh/docs/reference/glossary/dockershim.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+-->
+
+dockershim 是 Kubernetes v1.23 及之前版本中的一个组件，Kubernetes 系统组件通过它与 Docker Engine 通信。
+
+<!--more-->
+<!--
+Starting with version 1.24, dockershim has been removed from Kubernetes. For more information, see [Dockershim FAQ](/dockershim).
+-->
+从 Kubernetes v1.24 开始，dockershim 已从 Kubernetes 中移除.
+想了解更多信息，可参考 [移除 Dockershim 的常见问题](/zh/dockershim)。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/dockershim.md:35]:
```suggestion
想了解更多信息，可参考[移除 Dockershim 的常见问题](/zh/dockershim)。
```

File: content/zh/docs/reference/glossary/dockershim.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+short_description: >
+   A component of Kubernetes v1.23 and earlier, which allows Kubernetes system components to communicate with Docker Engine.
+-->
+
+<!--
+The dockershim is a component of Kubernetes version 1.23 and earlier. It allows the kubelet
+to communicate with {{< glossary_tooltip text="Docker Engine" term_id="docker" >}}.
+-->
+
+dockershim 是 Kubernetes v1.23 及之前版本中的一个组件，Kubernetes 系统组件通过它与 Docker Engine 通信。
```
Comment (Sea-n) [inline content/zh/docs/reference/glossary/dockershim.md:28]:
```suggestion
<!--
The dockershim is a component of Kubernetes version 1.23 and earlier. It allows the kubelet
to communicate with {{< glossary_tooltip text="Docker Engine" term_id="docker" >}}.
-->

dockershim 是 Kubernetes v1.23 及之前版本中的一个组件，Kubernetes 系统组件通过它与
{{< glossary_tooltip text="Docker Engine" term_id="docker" >}} 通信。
```

File: content/zh/docs/reference/glossary/dockershim.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: Dockershim
+id: dockershim
+date: 2022-04-15
+full_link: /dockershim
```
Comment (Sea-n) [inline content/zh/docs/reference/glossary/dockershim.md:5]:
```suggestion
full_link: /zh/dockershim
```

File: content/zh/docs/reference/glossary/dockershim.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+<!--
+title: Dockershim
+id: dockershim
+date: 2022-04-15
+full_link: /dockershim
+short_description: >
+   A component of Kubernetes v1.23 and earlier, which allows Kubernetes system components to communicate with Docker Engine.
+-->
```
Comment (Sea-n) [inline content/zh/docs/reference/glossary/dockershim.md:21]: Missed `aka` and `tags`
```suggestion
<!--
title: Dockershim
id: dockershim
date: 2022-04-15
full_link: /dockershim
short_description: >
   A component of Kubernetes v1.23 and earlier, which allows Kubernetes system components to communicate with Docker Engine.

aka:
tags:
- fundamental
-->
```

**Pull request merged**